posts/2021/oop-tutorial.md

```diff
@@ -0,0 +1,41 @@
+---
+author: Julia Kent
+date: 2021-11-08
+tags: python-tutorial-series
+---
+
+# Object Oriented Programming Tutorial
+
+The NCAR/UCAR virtual Python Tutorial Seminar Series continues with a lesson on Object Oriented Programming on **November 10th at 1 PM Mountain Standard Time** led by Dr. Kevin Paul.
+
+The content for this seminar tutorial is hosted on Kevin's [Object Oriented Programming Tutorial Repository](https://github.com/kmpaul/pg2oop).
+
+## Bio
+
+Dr. Kevin Paul is a computational physicist and software engineer working in the Computational & Information Systems Lab at the National Center for Atmospheric Research. Dr. Paul is the head of the Experimental Development Team (Xdev) and is passionate about open source development and teaching Python to Earth system scientists.
+
+## How-to-Run
+
+**This tutorial is designed to be launched from Binder.**
+
+- Click the "Launch Binder" button at the top level of the [repository](https://github.com/kmpaul/pg2oop).
+
+- Then, in the recently-launched Binder tab, select the notebook file `index.ipynb` to begin.
```

Comment:
You don't need this step.  You only need this step if you run locally from your laptop.

```suggestion
```